src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs

```diff
@@ -71,6 +74,13 @@ public async Task ConfigureAsync(ClusterConfig clusterConfig, PoolConfig poolCon
             extraPoolConfig = poolConfig.Extra.SafeExtensionDataAs<EthereumPoolConfigExtra>();
             extraConfig = poolConfig.PaymentProcessing.Extra.SafeExtensionDataAs<EthereumPoolPaymentProcessingConfigExtra>();
 
+            // if pKey and txEndpoint configured - setup web3 connection for self managed wallet payouts
+            if(!string.IsNullOrEmpty(extraConfig.PrivateKey) && !string.IsNullOrEmpty(extraConfig.TxEndpoint))
+            {
+                var account = new Account(extraConfig.PrivateKey);
+                web3Connection = new Nethereum.Web3.Web3(account, extraConfig.TxEndpoint);
```

Comment:
I would like to keep it try-catch to avoid connection issues unhandled. Also, swallow the error if it's an optional step.